core/perf_test/CMakeLists.txt

```diff
@@ -49,11 +49,19 @@ SET(SOURCES
   )
 
 IF(Kokkos_ENABLE_HIP)
-# FIXME requires TeamPolicy
+# FIXME HIP requires TeamPolicy
   LIST(REMOVE_ITEM SOURCES
     PerfTest_CustomReduction.cpp
     PerfTest_ExecSpacePartitioning.cpp
-    )
+  )
+ENDIF()
+
+IF(Kokkos_ENABLE_OPENMPTARGET)
+# FIXME OPENMPTARGET requires TeamPolicy Reductions and Custom Reduction
```

Comment:
You are mixing `FIXME OPENMPTARGET` with `WORKAROUND OPENMPTARGET`.  The point was to have unique token that can be searched for.

Reply:
I never used WORKAROUND in a cmake file, so I followed what was in the file.